### ISSUE TITLE
Heredoc is not processed.

### ISSUE THREAD

**Issue by IngwiePhoenix** [atom/language-php#113]
``` php
<?php
$str = <<<EOT
This is a foo.
    He likes bar.
        But bar likes baz.
EOT;
```

This is not recognized, but yet a simplified variant from the official PHP docs. Proper heredoc support is required since larger string sections in PHP are best done using that. For instance, I use it to store translated paragraphs.

**Comment by rugk** [comment]
There seem to be several [issues with Heredoc](https://github.com/atom/language-php/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+Heredoc+). Possibly this is a duplicate of https://github.com/atom/language-php/issues/10.

**Comment by IngwiePhoenix** [comment]
Yeah, looks like I am not the only one being unable to heredoc their stuff (:

Is a possible fix in sight?

**Comment by sbrl** [comment]
I don't think that this is a duplicate of #10, because heredoc isn't processed correctly even when it doesn't contain a closing PHP tag.

**Comment by 50Wliu** [comment]
Fixed by #184.
